R/FilteredDatasetDefault.R

```diff
@@ -1,269 +1,86 @@
-# DefaultFilteredDataset ------
+# DefaultFilteredDataset ----
+
 #' @title The `DefaultFilteredDataset` R6 class
+#' @description Stores any object as inert entity. Filtering is not supported.
 #' @keywords internal
 #' @examples
 #' library(shiny)
-#' ds <- teal.slice:::DefaultFilteredDataset$new(iris, "iris")
-#' ds$set_filter_state(
-#'   teal_slices(
-#'     teal_slice(dataname = "iris", varname = "Species", selected = "virginica"),
-#'     teal_slice(dataname = "iris", varname = "Petal.Length", selected = c(2.0, 5))
-#'   )
-#' )
+#' ds <- teal.slice:::DefaultFilteredDataset$new(letters, "letters")
 #' isolate(ds$get_filter_state())
 #' isolate(ds$get_call())
 DefaultFilteredDataset <- R6::R6Class( # nolint
   classname = "DefaultFilteredDataset",
   inherit = FilteredDataset,
+
+  ## Public Methods ----
```

Comment:
`ui_add` method is missing and should be implemented, otherwise a `teal` app fails

Reply:
I forgot to check in an app :sweat_smile:

Reply:
Added `ui_add` and `ui_active`, the latter in order to NOT display the dataset name in the filter panel.